.github/workflows/ci.yml

```diff
@@ -132,23 +132,9 @@ jobs:
     container:
       image: "kiwix/kiwix-build_ci:${{matrix.image_variant}}-32"
     steps:
-    - name: Checkout code
-      shell: python
-      run: |
-        from subprocess import check_call
-        from os import environ
-        config_command = [
-          'git', 'config', '--global',
-          'http.postBuffer', '1048576000'
-        ]
-        check_call(config_command, cwd=environ['HOME'])
-        clone_command = [
-          'git', 'clone',
-          'https://github.com/${{github.repository}}',
-          '--depth=1',
-          '--branch', '${{github.ref_name}}'
-        ]
-        check_call(clone_command, cwd=environ['HOME'])
+      - name: Checkout code
```

Comment:
Would be nice to have a explanation in the commit message of why we can now use the github action instead of the custom git clone.

Reply:
fixed